### PR TITLE
GH actions. build-packages.yml: check if hash entries were created for all targets

### PR DESCRIPTION
**Pull request by romeroalx**
### Short description
Check if `pkghashes` and `srchashes` entries were created for all targets in `build-packages.yml`.

### Checklist

I have:
- [ ] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [ ] compiled this code
- [x] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)
- [ ]  checked that this code was merged to master

### CODE REVIEW

File: .github/workflows/build-packages.yml [hunk 1]
```diff
@@ -136,6 +136,18 @@ jobs:
           echo "$HOSTKEY" > ~/.ssh/known_hosts
           rsync -4rlptD built_pkgs/* "$RSYNCTARGET"
 
+  check-hashes:
+    needs: build
+    name: Check if hashes were created for all requested targets
+    runs-on: ubuntu-20.04
```
Comment (Habbie) [inline .github/workflows/build-packages.yml:142]: might as well do 22.04 for new jobs

**Review by Habbie — APPROVED** [review]
LGTM, one small nit. Would like Rémi's review too.

**Review by rgacogne — APPROVED** [review]
Nice!

File: .github/workflows/build-packages.yml (hunk 1 above)
Comment (rgacogne) [reply .github/workflows/build-packages.yml:142]: Do we want to change this to 22.04 before merging?

Comment (romeroalx) [reply .github/workflows/build-packages.yml:142]: I wanted to create a new PR once this is merged for upgrading all jobs in this workflow to 22.04 to have all of them on the same version.

Comment (rgacogne) [reply .github/workflows/build-packages.yml:142]: Alright, I'll merge this PR now then :)

**Pull request merged**